lib/rules/no-get.js

```diff
@@ -153,19 +177,20 @@ module.exports = {
           const path = node.arguments[1].value;
           context.report({
             node,
-            message: makeErrorMessageForGet(path, true),
+            message: makeErrorMessageForGet(path, true, useOptionalChaining),
             fix(fixer) {
-              if (path.includes('.')) {
+              if (path.includes('.') && !useOptionalChaining) {
```

Comment:
This fix for both paths seems to be identical. Should it be extracted?

Reply:
Good point, extracted.